docs/administration/configure-authentication/db-auth.md

```diff
@@ -13,16 +13,16 @@ If you create users in the database, Harbor is locked in database mode. You cann
 1. Under **Administration**, go to **Configuration** and select the **Authentication** tab.
 1. Leave **Auth Mode** set to the default **Database** option.
 
-   ![Database authentication](../../../img/db-auth.png)
```

Comment:
The images are linked this way because they are supposed to be viewed on https://goharbor.io, not here on GitHub.